bool zero_gyros
bool reset_ekf
bool set_mag_ref
---
